### PR TITLE
Add quiz spec to examples/ directory

### PR DESCRIPTION
**Pull request by rickcnagy**
This is what we have so far for definining quiz versions. We likely also
want to do something to define quizzes (i.e. qwiklabs.yaml in
examples/quiz) but we haven't figured that out yet.

### CODE REVIEW

File: examples/quiz/en.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+schema_version: 1
+
+locale: en
```
Comment (jgan) [inline examples/quiz/en.yaml]: Having one locale per .yml file is cool for the upload interface and the GitWhisperer implementation, but it's contrary to the content bundle design specs.

Check out the example lab's use of default locale:

https://github.com/CloudVLab/qwiklabs-content-bundle-spec/blob/3cdcd1909acff93a1617836273e89612dfcb40bd/examples/lab-robust/qwiklabs.yaml#L5

... and localizable properties:

https://github.com/CloudVLab/qwiklabs-content-bundle-spec/blob/3cdcd1909acff93a1617836273e89612dfcb40bd/examples/lab-robust/qwiklabs.yaml#L7-L15

Comment (jgan) [reply examples/quiz/en.yaml]: @rickcnagy - poke

**Comment by rickcnagy** [comment]
@jgan - I’ve been thinking about this, let’s discuss tomorrow...

**Comment by rickcnagy** [comment]
Closing in favor of #15 .

**Pull request closed without merging**